### ISSUE TITLE
Issue with character escaping when using Prism

### ISSUE THREAD

**Issue by apexJCL** [glorious-codes/glorious-demo#54]
Assuming the following code:

```js
const gdemo = new GDemo('[data-demo-container]');

const code = `
let x = 4

if (x > 2) {
  console.log('Oops')
}
`

const highlightedCode = Prism.highlight(
  code,
  Prism.languages.javascript,
  'javascript'
);

gdemo
  .openApp('editor', { minHeight: '400px', windowTitle: 'demo.js' })
  .write(code)
  .write(highlightedCode, { onCompleteDelay: 2000 })
  .end();
```

Using **Prism** for highlight, makes glorious-demo produce incorrect characters, replacing the `>` for `\&gt\;`.
![glorious_issue](https://user-images.githubusercontent.com/4037632/52655666-5072b400-2eba-11e9-96f9-42549842097e.png)

**Comment by rafaelcamargo** [comment]
Thanks for reporting the bug, José!

As soon as I get some free time, I'll fix it.

**Comment by apexJCL** [comment]
Alright, as I've seen, you have a call [in here](https://github.com/glorious-codes/glorious-demo/blob/master/src/scripts/services/type-html-text/type-html-text.js#L29) when the node is not text, then the issue is [over here](https://github.com/glorious-codes/glorious-demo/blob/master/src/scripts/services/type-plain-text/type-plain-text.js#L3), because the method it's receiving the escaped string, `=&amp;`, then it's getting split and appended.

I think that's the issue, I don't know, just gave a brief overlook to the project, but it's a nice project overall.
Thanks!

**Comment by rafaelcamargo** [comment]
@apexJCL Hey man, fixed on v0.8.1. Thanks for letting me know this bug ✌️
